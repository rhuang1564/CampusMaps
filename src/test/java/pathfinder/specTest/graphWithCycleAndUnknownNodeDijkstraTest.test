# Find a path to nonexistant node with Dijkstra's algorithm
# with a cycle in the graph

CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C
AddNode graph D

AddEdge graph A B 1.0
AddEdge graph B C 1.0
AddEdge graph D A 1.0
AddEdge graph C A 1.0
AddEdge graph C D 10.0

FindPath graph A E

FindPath graph E A

FindPath graph E E
