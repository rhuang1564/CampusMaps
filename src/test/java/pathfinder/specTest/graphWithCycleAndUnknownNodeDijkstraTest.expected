# Find a path to nonexistant node with Dijkstra's algorithm
# with a cycle in the graph

created graph graph

added node A to graph
added node B to graph
added node C to graph
added node D to graph

added edge 1.000 from A to B in graph
added edge 1.000 from B to C in graph
added edge 1.000 from D to A in graph
added edge 1.000 from C to A in graph
added edge 10.000 from C to D in graph

unknown node E

unknown node E

unknown node E
unknown node E